Sources/TokamakCore/Animation/Animation.swift

```diff
@@ -0,0 +1,215 @@
+// Copyright 2020 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+
+public let defaultDuration = 0.35
```

Comment:
Would a comment here that describes how this value was inferred make sense? Or just a reassuring sentence that this was visually tested to match the duration in SwiftUI on Apple platforms?